.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "OSS/markdowndeep"]
 	path = OSS/markdowndeep
-	url = https://github.com/OneDrive/markdowndeep.git
+	url = https://github.com/baywet/markdowndeep.git
```

Comment:
Do we have a tracking work item to update this reference to use a shared version of markdowndeep?

Reply:
I don't think we need a work item to track that, I'll revert that change once my PR on markdowndeep gets merged, see https://github.com/OneDrive/markdowndeep/pull/4